src/SFA.DAS.EAS.Portal/Application/Commands/ReserveFunding/AddReserveFundingCommand.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.CosmosDb;
+using SFA.DAS.EAS.Portal.Database;
+using SFA.DAS.EAS.Portal.Database.Models;
+using SFA.DAS.Reservations.Messages;
+
+namespace SFA.DAS.EAS.Portal.Application.Commands
+{
+    public class AddReserveFundingCommand //: IAddReserveFundingCommand
```

Comment:
Is the commented out interface intentional?

Reply:
It's been deleted.